Xamarin.Forms.Platform.iOS/ContextScrollViewDelegate.cs

```diff
@@ -157,7 +157,8 @@ public override void WillEndDragging(UIScrollView scrollView, PointF velocity, r
 							IsOpen = false;
 							scrollView.SetContentOffset(new PointF(0, 0), true);
 
-							ClearCloserRecognizer(scrollView);
+							if (_globalCloser.State != UIGestureRecognizerState.Cancelled)
```

Comment:
If the state is Canceled dont we still need to dispose it ? you should make this check inside the ClearCloserRecognizer and only where it's throwing the ex.